src/plugins/imagery/mixins/imageryData.js

```diff
@@ -55,8 +57,23 @@ export default {
 
         this.stopFollowingDataTimeContext();
         this.openmct.objectViews.off('clearData', this.clearData);
```

Comment:
How does `clearData` interface with telemetryCollections? I see that you've removed this method. Does clearing data still work?

Reply:
Good catch! I've added it back

Reply:
Hadn't realized telemetryCollection's clear was different